We do not want a bundle with wxGTK in this case.
(It might be possible to still build this as a bundle,
but it will require more patches to the sources. No need.)

--- configure	2024-04-16 02:20:29.000000000 +0800
+++ configure	2024-04-27 22:26:10.000000000 +0800
@@ -21311,7 +21311,7 @@
   # ------------------------
 
   # Create application bundle on OS X
-  if test "$mac" = "1"; then
+  if echo $WX_CPPFLAGS | grep __WXMAC__ > /dev/null 2>&1; then
     macappbundle="yes"
   else
     macappbundle="no"

--- src/commonui/Makefile.am	2023-05-10 16:01:51.000000000 +0800
+++ src/commonui/Makefile.am	2024-04-24 09:31:15.000000000 +0800
@@ -65,8 +65,6 @@
 libfzclient_commonui_private_la_LDFLAGS += -lole32 -Wl,-luuid
 endif
 
-if MACAPPBUNDLE
-
 noinst_LTLIBRARIES = libfzclient-commonui-private-mac.la
 
 libfzclient_commonui_private_mac_la_CPPFLAGS = -I$(top_builddir)/config
@@ -84,5 +82,3 @@
 
 libfzclient_commonui_private_la_LIBADD = libfzclient-commonui-private-mac.la
 libfzclient_commonui_private_la_DEPENDENCIES += libfzclient-commonui-private-mac.la
-
-endif
